ledger/participant-state/src/main/scala/com/daml/ledger/participant/state/v2/package.scala

```diff
@@ -45,26 +43,4 @@ import com.daml.lf.data.Ref
   * participant state APIs at the same time. We plan to deprecate and drop old
   * versions on separate and appropriate timelines.
   */
-package object v2 {
-
-  /** Identifiers for transactions. */
-  val TransactionId: Ref.LedgerString.type = Ref.LedgerString
-  type TransactionId = Ref.LedgerString
-
-  /** Identifiers used to correlate commands with results. */
-  val CommandId: Ref.LedgerString.type = Ref.LedgerString
-  type CommandId = Ref.LedgerString
-
-  /** Identifiers used for correlating submission with a workflow. */
-  val WorkflowId: Ref.LedgerString.type = Ref.LedgerString
-  type WorkflowId = Ref.LedgerString
-
-  /** Identifiers for submitting client applications. */
-  val ApplicationId: Ref.LedgerString.type = Ref.LedgerString
-  type ApplicationId = Ref.LedgerString
-
-  /** Identifiers used to correlate submissions with results. */
-  val SubmissionId: Ref.LedgerString.type = Ref.LedgerString
-  type SubmissionId = Ref.LedgerString
-
-}
+package object v2
```

Comment:
does this still make sense to have it empty?

Reply:
See Fabio's comment.